src/Card/Card.js

```diff
@@ -0,0 +1,206 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import { FlexBox } from "../FlexBox/FlexBox";
+import { Heading } from "../Heading/Heading";
+import { cardAlignOptions } from "./Card.levers";
+
+/**
+ * Card Component
+ *
+ * HTML card element
+ */
+
+export const Card = ({ className, children, isMinimal, ...props }) => {
+  // Defaults & Variables.
+  // ---------------------------------------------------------------------------
+
+  let wrapperClasses =
+    "su-bg-white su-text-black su-border su-border-solid su-border-black-10 su-shadow";
+
+  if (isMinimal) {
+    wrapperClasses = "";
+  }
+
+  return (
+    <article
+      className={dcnb(
+        "basic-card su-max-w-600 su-basefont-23",
+        wrapperClasses,
+        className
+      )}
+    >
+      {children}
+    </article>
+  );
+};
+
+// Subcomponents
+// ---------------------------------------------------------------------------
+const Content = ({ children, className, align }) => {
+  // Content alignment including image and CTA, default is left-align
+  // This setting overrides the alignment option in the nested CTA
+  let bodyAlign = "su-items-start";
+  let contentClass = "";
+
+  if (align === "center") {
+    contentClass = dcnb(contentClass, "children:su-mx-auto su-text-center");
+    bodyAlign = "su-items-center";
+  }
+
+  return (
+    <FlexBox
+      direction="col"
+      className={dcnb("card-body", bodyAlign, contentClass, className)}
+    >
+      {children}
+    </FlexBox>
+  );
+};
+
+Content.displayName = Content;
+Card.Content = Content;
+
+const Image = ({ image, className }) => {
```

Comment:
It doesn't make sense to make image an object here if you are only using one property. Two paths forward for you. 

1. Continue to use image as an object and destructure what you want out of it for `src` but then expand the rest of the props on the `img` element so users can modify the values. 
2. Change the image prop to a string type and rename `filename`

Reply:
You destructure the className prop but don't use it anywhere.